dbms/src/Interpreters/AsynchronousMetrics.cpp

```diff
@@ -177,6 +182,32 @@ void AsynchronousMetrics::update()
         set("MaxDTBackgroundTasksLength", max_dt_background_tasks_length);
     }
 
+    do
```

Comment:
can we add a method to replace the sub logical?

Reply:
done